spread/qemu.go

```diff
@@ -118,6 +118,10 @@ func (p *qemuProvider) Allocate(ctx context.Context, system *System) (Server, er
 	if os.Getenv("SPREAD_QEMU_GUI") != "1" {
 		cmd.Args = append([]string{cmd.Args[0], "-nographic"}, cmd.Args[1:]...)
 	}
+	if biosPath := os.Getenv("SPREAD_QEMU_BIOS"); biosPath != "" {
+		cmd.Args = append([]string{cmd.Args[0], "-bios", biosPath}, cmd.Args[1:]...)
+	}
```

Comment:
We may need to use `-pflash` later when working with secure boot to be able to enroll our keys and certificates to our nvram file (which we'll also need to specify).